submissions/Nik3264/oop-exercise/index.js

```diff
@@ -0,0 +1,163 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: _https://github.com/Nik3264/OOP-exercise_
+   Web app: _https://nik3264.github.io/OOP-exercise/_
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+
+class Animal {
+  constructor(species, name, gender, legs, saying) {
```

Comment:
You had constructor's parameters list nicely arranged in an object deconstruction syntax.
Btw, did you know object deconstruction syntax allows for default values?